lib/framereader.py

```diff
@@ -165,6 +165,28 @@ def read_frame_P210(fd, width, height):
 
   return y, u, v
 
+def read_frame_Y210(fd, width, height):
+  #https://docs.microsoft.com/en-us/windows/win32/medfound/10-bit-and-16-bit-yuv-video-formats#422-formats
+  #Y210 (each pair of pixels is stored as an array of four WORD)
+  #   Y0           U            Y1             V
+  #  word0        word1        word2         word3
+  #bits 15-0     bits15-0     bits15-0      bits15-0
+  #
+  #The 16bit representations described here use little-endian WORD values for each channel
+  #bits:  15 14 13 ...  ...     5 4 3 2 1 0
+  #data:  valid data  ...       0 0 0 0 0 0
+  width2  = (width + 1) // 2
+  size    = width * height
+  size2   = width2 * height * 4
+
+  y210 = numpy.fromfile(fd, dtype=numpy.uint16, count=size2)
+  y0 = y210[0::4].reshape((height, width2)) & 0xffc0
+  u  = y210[1::4].reshape((height, width2)) & 0xffc0
+  y1 = y210[2::4].reshape((height, width2)) & 0xffc0
+  v  = y210[3::4].reshape((height, width2)) & 0xffc0
+
+  return y0, u, y1, v
```

Comment:
We should only return y, u and v arrays so they can be easily compared with frames from other formats.

That is, `y = y210[0::2]`